packages/lesswrong/components/ultraFeed/useSeeLess.ts

```diff
@@ -72,6 +82,81 @@ export const useSeeLess = ({ documentId, documentType, recommId }: UseSeeLessOpt
     setSeeLessEventId(null);
   }, [currentUser, seeLessEventId, documentId, documentType, recommId, updateUltraFeedEvent, captureEvent]);
 
+  const handleSeeLessClick = useCallback(() => {
+    void (async () => {
+      if (!currentUser) return;
+
+      if (isSeeLessMode) {
+        void handleUndoSeeLess();
+        return;
+      }
+
+      handleSeeLess('pending');
```

Comment:
I think I didn't catch this the first time around, but... this is really not the way to manage state.  Instead of overloading `handleSeeLess` to handle both updating seeLessMode and seeLessEventId (by sometimes passing in an id and sometimes passing in `pending`), this should just directly call `setSeeLessMode(true)`.